docs/docsite/rst/porting_guides/porting_guide_2.11.rst

```diff
@@ -37,8 +37,22 @@ No notable changes
 Modules
 =======
 
-No notable changes
+Change to Default File Permissions
+----------------------------------
+
+To address CVE-2020-1736, the default permissions for certain files created by Ansible using ``atomic_move()`` were changed from ``0o666`` to ``0o600``. The default permissions vaule was only used for the temporary file before it was moved into its place or newly created files. If the file existed when the new temporary file was moved into place, Ansible would use the permissions of the existing file. If there was no existing file, Ansible would retain the default file permissions of the temporary file.
```

Comment:
This should be clarified to mention that umask is applied to the old default, since that will in most cases result in a more restrictive default.